frontend/javascripts/oxalis/view/action-bar/starting_job_modals.tsx

```diff
@@ -348,10 +370,99 @@ export function OutputSegmentationLayerNameFormItem({
+      title={
+        <>
+          <i className="fas fa-magic" />
+          Automated analysis with WEBKNOSSOS
+        </>
+      }
+      onCancel={() => Store.dispatch(setAIJobModalStateAction("invisible"))}
+      footer={null}
+    >
+      <Space direction="vertical" size="middle">
+        <Row>Choose a processing job for your dataset:</Row>
+        <Space align="center">
+          <Radio.Button
+            className="aIJobSelection"
+            checked={aIJobModalState === "neuron_inferral"}
+            onClick={() => Store.dispatch(setAIJobModalStateAction("neuron_inferral"))}
+          >
+            <Card bordered={false}>
+              <Space direction="vertical" size="small">
+                <Row>Neuron segmentation</Row>
+                <Row>
+                  <img
+                    src={`/assets/images/${jobNameToImagePath.neuron_inferral}`}
+                    alt={"Neuron segmentation example"}
+                    style={centerImageStyle}
+                  />
+                </Row>
+              </Space>
+            </Card>
+          </Radio.Button>
+          <Tooltip title="Coming soon">
+            <Radio.Button
+              className="aIJobSelection"
+              disabled
+              checked={aIJobModalState === "nuclei_inferral"}
+              onClick={() => Store.dispatch(setAIJobModalStateAction("nuclei_inferral"))}
+            >
+              <Card bordered={false}>
+                <Space direction="vertical" size="small">
+                  <Row>Nuclei detection</Row>
+                  <Row>
+                    <img
+                      src={`/assets/images/${jobNameToImagePath.nuclei_inferral}`}
+                      alt={"Nuclei inferral example"}
+                      style={centerImageStyle}
+                    />
+                  </Row>
+                </Space>
+              </Card>
+            </Radio.Button>
+          </Tooltip>
+          <Tooltip title="Coming soon">
+            <Radio.Button
+              className="aIJobSelection"
+              disabled
+              checked={aIJobModalState === "mitochondria_inferral"}
+              onClick={() => Store.dispatch(setAIJobModalStateAction("mitochondria_inferral"))}
+            >
+              <Card bordered={false}>
+                <Space direction="vertical" size="small">
+                  <Row>Mitochondria detection</Row>
+                  <Row>
+                    <img
+                      src={`/assets/images/${jobNameToImagePath.mitochondria_inferral}`}
+                      alt={"Mitochondria detection example"}
+                      style={centerImageStyle}
+                    />
+                  </Row>
+                </Space>
+              </Card>
+            </Radio.Button>
+          </Tooltip>
+        </Space>
+        {aIJobModalState === "neuron_inferral" ? <NeuronSegmentationForm /> : null}
+        {aIJobModalState === "nuclei_inferral" ? <NucleiSegmentationForm /> : null}
```

Comment:
Although it has been like this before, could you please rename `NucleiSegmentationForm` to `NucleiDetectionForm`? The segmentation part is misleading otherwise.